homeassistant/components/airzone/entity.py

```diff
@@ -67,6 +68,33 @@ def get_airzone_value(self, key: str) -> Any:
         return value
 
 
+class AirzoneWebServerEntity(AirzoneEntity):
+    """Define an Airzone WebServer entity."""
+
+    def __init__(
+        self,
+        coordinator: AirzoneUpdateCoordinator,
+        entry: ConfigEntry,
+    ) -> None:
+        """Initialize."""
+        super().__init__(coordinator)
+
+        mac = self.get_airzone_value(AZD_MAC)
+        self._attr_device_info: DeviceInfo = {
+            "connections": {(dr.CONNECTION_NETWORK_MAC, mac)},
+            "identifiers": {(DOMAIN, f"{entry.entry_id}_ws")},
+            "manufacturer": MANUFACTURER,
+            "model": self.get_airzone_value(AZD_MODEL),
+            "name": self.get_airzone_value(AZD_FULL_NAME),
+            "sw_version": self.get_airzone_value(AZD_FIRMWARE),
+        }
```

Comment:
Since `DeviceInfo` is a `TypedDict` you can do
```
DeviceInfo(
  connection={(dr.CONNECTION_NETWORK_MAC, mac)},
  ...
)
```

Reply:
Done in d833a6d2d08e1e40a7ab2f7170c435ea2e81497b